library/src/scala/internal/tasty/CompilerInterface.scala

```diff
@@ -33,9 +33,9 @@ trait CompilerInterface extends scala.tasty.reflect.Types {
   // Constraints //
   /////////////////
 
-  def Constraints_init(self: Context): Context
-  def Constraints_add(self: Context)(syms: List[Symbol]): Boolean
-  def Constraints_approximation(self: Context)(sym: Symbol, fromBelow: Boolean): Type
+  def Constraints_context[T]: scala.quoted.QuoteContext
+  def Constraints_add(syms: List[Symbol]): Boolean
+  def Constraints_approximation(sym: Symbol, fromBelow: Boolean): Type
```

Comment:
Wow, I'm very surprised that we're exposing this stuff in the public tasty-reflect API. If this is only used internally could it be `private[scala]` at least?

Reply:
We are not. The CompilerInterface API is not accessible from user code. The `Reflection` mixes in this class without exposing its members.